server/src/test/resources/reference.conf

```diff
@@ -1,8 +1,10 @@
+server-port=8080
 working-directory = "/tmp"
 is-rstudio-runtime = false
 type = "azure"
 misc-http-client-config = {
   wsm-url = "https://workspace.dsde-dev.broadinstitute.org/"
   workspace-id = "a5a1f1e1-bcb0-49d9-b589-ea4d7c9d6f02"
-  storage-container-resource-id = "9151f3a0-fe5c-49c5-b8a1-dc15cd96b174"
-}
\ No newline at end of file
+}
+workspace-storage-container-resource-id = "9151f3a0-fe5c-49c5-b8a1-dc15cd96b174"
```

Comment:
are these stable entities that exist in the cloud and tests depend on, or dummy values that are never hit because the relevant calls are mocked? Probably worth making a comment to indicate which in the file.

Reply:
these values are mostly just for `ConfigSpec`...no unit test is directly calling any external service..so the actual value doesn't really matter